plugfest/index.html

```diff
@@ -0,0 +1,239 @@
+          font-size: 17px;
+          padding: 0 5px;
+          text-decoration-line: none;
+        }
+        
+        footer p {
+          color: white;
+          font-family: 'Open Sans', sans-serif;
+          font-size: 14px;
+          padding-top: 12px;
+        }
+        
+        form {
+          border: solid 1px black;
+          margin-top: 5px;
+          margin-bottom: 5px;
+          padding-left: 10px;
+          padding-right: 10px;
+        }
+        
+        .form-control {
+          width: 100%;
+          text-align: right;
+        }
+        
+        table {
+          width: 100%;
+          text-align: center;
+        }
+        
+        textarea {
+          resize: vertical;
+          width: calc(100% - 2*3px);
+        }
+
+        .btn {
+          background: rgb(51, 184, 164);
+          border: 1px solid transparent;
+          border-radius: 200px;
+          color: white;
+          padding: 12px 25px;
+          font-size: 16px;
+          cursor: pointer;
+        }
+        
+        .highlighted {
+          text-align: center;
+          font-style: italic;
+        }
+
+        .framed {
+          padding: 12px 25px;
+          font-size: 16px;
+        }
+        
+        .framed a {
+          color: white;
+        }
+        
+        p.success {
+          background: rgb(51, 184, 164);
+          color: white;
+        }
+        
+        p.failure {
+          background: rgb(161, 33, 33);
+          color: white;
+        }
+        
+        table.success tr {
+          background: rgb(87, 109, 132);
+          color: white;
+        }
+        
+        table.failure tr {
+          background: rgb(161, 33, 33);
+          color: white;
+        }
+      </style>
+    </head>
+    <body>
+      <header>
+        <div class="container">
+          <img alt="Thingweb logo" src="./img/logo.png">
+        </div>
+      </header>
+    
+      <div class="wrapper">
+	  
+<h1>Services to support the W3C WoT Plugfests</h1>
+
+<p>We will provide sevices to support the operation of a Plugfest of W3C WoT</p>
+
+<!-- [DP] IAM Services not running anymore -->
+<!--
+<h2>IAM Services</h2>
+<p>See the <a href="http://w3c.github.io/wot/current-practices/wot-practices.html#security-mechanisms" target="_blank">security mechanisms</a> described in the Current Practices document. This service currently offers OAuth tokens using RS256 signatures.</p>
+<ul>
+    <li>Token endpoint:<br/> <a href="https://plugfest.thingweb.io:8443/wos/0.1/oauth/token">https://plugfest.thingweb.io:8443/wos/0.1/oauth/token</a></li>
+    <li>Registration endpoint:<br/> <a href="https://plugfest.thingweb.io:8443/wos/0.1/oauth/register">https://plugfest.thingweb.io:8443/wos/0.1/oauth/register</a> </li>
+    <li>Signing Key (RS256):<br/> <a href="https://plugfest.thingweb.io:8443/wos/0.1/oauth/jwk">https://plugfest.thingweb.io:8443/wos/0.1/oauth/jwk</a></li>
+</ul>
+-->
+
+
+
+<h2>TD Directory</h2>
```

Comment:
Do you want to update the content later and backup the status quo first?

Reply:
I will update it later so that we can see the actual change